chart/application/templates/_helpers.tpl

```diff
@@ -24,10 +24,21 @@ app.kubernetes.io/name: {{ .Values.epinio.appName }}
 app.kubernetes.io/component: application
```

Comment:
TODO: We'll need to bump this chart's version in order to create a new release. Then we will need to reference that new version in epinio.